sdks/java/core/src/test/java/org/apache/beam/sdk/values/RowTest.java

```diff
@@ -174,6 +176,24 @@ public void testCreatesArrayArray() {
     assertEquals(data, row.getArray("array"));
   }
 
+  @Test
+  public void testCreatesMap() {
+    Map<Integer, String> data = new HashMap<Integer, String>() {
+      {
+        put(1, "value1");
+        put(2, "value2");
+        put(3, "value3");
+        put(4, "value4");
```

Comment:
<!--new_thread; commit:7bf9d2048e8d568ee9280629db391775cc490358; resolved:0-->
I would also add a test where the map value is itself a complex or nested type.

Reply:
will update to support primitive/array/map/row as value type in Map